client/blocks/login/login-form.jsx

```diff
@@ -13,6 +13,7 @@ import { localize } from 'i18n-calypso';
  * Internal dependencies
  */
 import config from '@automattic/calypso-config';
+import { addQueryArgs } from '@wordpress/url';
```

Comment:
I could be wrong here, but for some reason I think using `@wordpress/url` seems to cause the canary e2e test failure. I don't know the exact details as to why, but I think using `{ addQueryArgs } from 'calypso/lib/url'` will bring canary e2e test back to passing.   p1612388346015200/1612387066.012000-slack-C1A1EKDGQ